crates/chain/src/chain_graph.rs

```diff
@@ -210,10 +212,10 @@ where
     ///
     /// This does not necessarily mean that it is *confirmed* in the blockchain; it might just be in
     /// the unconfirmed transaction list within the [`SparseChain`].
-    pub fn get_tx_in_chain(&self, txid: Txid) -> Option<(&P, &Transaction)> {
+    pub fn get_tx_in_chain(&self, txid: Txid) -> Option<(&P, TxInGraph<'_, Transaction, BlockId>)> {
```

Comment:
why change this API?

Reply:
Reverted in cfee20463c8468b99c97e0e73c8274e39173ad80